src/web/WebPlugin.ts

```diff
@@ -25,11 +26,12 @@ const activeRequestsGauge = new ExtendedGauge({
 
 export class RouteRegisterUtil {
   private routesToRegister = [];
-  private express: e;
+  private express;
 
   doRegister() {
     this.routesToRegister.forEach((route: {verb: string, path: string, instanceProperty: string, methodName: string, objectName: string}) => {
       logger.info(`Registering route from ${route.objectName}: ${route.verb.toUpperCase()} ${route.path} -> ${route.methodName}`);
+      this.express = e();
```

Comment:
@wz185 ... here you're overwriting the express instance that was passed to this in line 109. Any reason for it?